url/Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["parser-implementations", "web-programming", "encoding"]
 license = "MIT OR Apache-2.0"
 include = ["src/**/*", "LICENSE-*", "README.md", "tests/**"]
 edition = "2018"
-rust-version = "1.56"
+rust-version = "1.60"
```

Comment:
Probably should update the other crates as well?

(On a phone rn, tricky to check if they have this already. Probably should)

Reply:
Will check right after dinner.